src/runtime_src/core/common/api/xrt_hw_context.cpp

```diff
@@ -40,6 +42,7 @@ class hw_context_impl : public std::enable_shared_from_this<hw_context_impl>
     , m_cfg_param(cfg_param)
     , m_mode(xrt::hw_context::access_mode::shared)
     , m_hdl{m_core_device->create_hw_context(xclbin_id, m_cfg_param, m_mode)}
+    , m_usage_logger(xrt_core::usage_metrics::get_usage_metrics_logger())
```

Comment:
You could use default member initialization since no constructor arguments are relied on.